### PR TITLE
[CVE-2017-8418] - updating rubocop dependency.

### PR DESCRIPTION
**Pull request by majormoses**
Breaking Changes:
- removed ruby `< 2.1` support

Misc:
- appeased the cops
- fixed bug with single quotes where interpolation is needed
- update changelog guidelines link

Signed-off-by: Ben Abrams <me@benabrams.it>

## Pull Request Checklist

https://github.com/sensu-plugins/community/issues/77

#### General

- [ ] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)

- [ ] Update README with any necessary configuration snippets

- [ ] Binstubs are created if needed

- [ ] RuboCop passes

- [ ] Existing tests pass

#### Purpose
Resolve CVE, see parent issue for details

#### Known Compatibility Issues
Removes ruby `< 2.1` support

### CODE REVIEW

File: bin/metrics-xen.rb [hunk 1]
```diff
@@ -39,14 +39,15 @@ class XenGraphite < Sensu::Plugin::Metric::CLI::Graphite
...
+         default: "#{Socket.gethostbyname(Socket.gethostname).first}.xen"
 
   def run
     xen = metrics_hash
     xm = xm_hash
     xencount = xenservers
     i = 0
-    xen.each do |_k, v|
+    # TODO: come back and refactor me
+    xen.each_value do |_k, v| # rubocop:disable Metrics/BlockLength
```
Comment (amdprophet) [inline bin/metrics-xen.rb]: I think `each_value` only passes one param (the value of each key) into the block.

Comment (majormoses) [reply bin/metrics-xen.rb]: Good catch, was too quick to look at the rubocop correction closely enough. I will get that fixed up.

**Comment by majormoses** [comment]
@amdprophet should be good now, let me know if you spot anything else.

**Review by jaredledvina — APPROVED** [review]
LGTM

**Comment by majormoses** [comment]
released: https://rubygems.org/gems/sensu-plugins-xen/versions/3.0.0

**Pull request merged**